app/models/article.rb

```diff
@@ -19,7 +19,6 @@ class Article < ApplicationRecord
   delegate :username, to: :user, prefix: true
 
   belongs_to :user
-  belongs_to :job_opportunity, optional: true
```

Comment:
Do you know if having this code in place without the table will cause any problems when we try to save an article?

Reply:
If we leave the optional relation? No, AFAIK Rails will only complain if the relation is not optional. You can add `belongs_to :whatever, optional: true` and Rails will ignore it during save. Or at least that's what I just tried to do :D

Reply:
that works then!